modules/svg/doc_classes/ResourceImporterLottie.xml

```diff
@@ -0,0 +1,125 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="ResourceImporterLottie" inherits="ResourceImporter" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../../doc/class.xsd">
+	<brief_description>
+		Import a Lottie animation as a sprite sheet.
+	</brief_description>
+	<description>
+		This imports a Lottie animation to a [CompressedTexture2D]. See also [ResourceImporterTexture].
+		[b]Note:[/b] Both JSON and dotLottie (extra resources are ignored) are supported. Only files with the [code].lottie[/code] extension are recognized. You will have to rename the JSON file extension to [code].lottie[/code] to import it.
```

Comment:
Just as a heads up, I'm part of the working group that is standardizing the Lottie format and we are considering using `.lot` as recommended file extension for the JSON files so it could be useful to add support for it as well (we're finalizing the IANA registration)